src/txacme/client.py

```diff
@@ -771,6 +783,17 @@ def _send_request(self, method, url, *args, **kwargs):
                             b'content-type', [None])[0])))
                 .addActionFinish())
 
+    def stop(self):
+        """
+        Stops the operation.
+
+        This cancels pending operations and does cleanup.
```

Comment:
This isn't actually quite correct; active requests will still proceed, only idle keep-alive connections are closed.

Reply:
True...but I think that this is what the API should do :)

So the docstring is correct, is just that the implementation is broken,

As commented before, it would be nice to have an Agent.shutdown() API, or something like that which can be used to instruct an HTTP client agent to stop and cancel everything it is doing

Reply:
Yes, that would be nice :)